Sources/HaCWebsiteLib/routes.swift

```diff
@@ -20,10 +20,14 @@ func getWebsiteRouter() -> Router {
   router.all("/static", middleware: StaticFileServer(path: "./static/dist"))
 
   /// Intended for use by GitHub webhooks
-  router.post("/api/refresh_workshops", handler: GitHubWebhookController.handler)
+  // TODO: make the GitHubWebhookController configurable!
```

Comment:
What more configurability do we want?

Reply:
That TODO was made before I made the configurability changes, my bad!